### PR TITLE
ADR-1292 - Added 'Remove' product functionality

### PR DESCRIPTION
**Pull request by chiransan01**

### CODE REVIEW

File: src/test/resources/features/alcoholToDeclareJourney/AlcoholDutyReturnFullJourney.feature [hunk 1]
```diff
@@ -1361,4 +1361,85 @@ Feature: Alcohol Duty Returns Journey
       | Adjustment duty value | Nil | Change |
     When I click on Agree and send return button "Return Summary Page"
     Then I am presented with the "Return Submitted Page"
-    And the page source contains "You have nothing to pay"
\ No newline at end of file
+    And the page source contains "You have nothing to pay"
+
+  Scenario:4. Alcohol Duty Returns Journey - Verifying 'Remove' link functionality
```
Comment (Kavitakum) [inline src/test/resources/features/alcoholToDeclareJourney/AlcoholDutyReturnFullJourney.feature:1366]: Are we not adding @Test tag to this test case??

Comment (chiransan01) [reply src/test/resources/features/alcoholToDeclareJourney/AlcoholDutyReturnFullJourney.feature:1366]: No, we added at the feature level so you don't need one at the scenario level

Comment (Kavitakum) [reply src/test/resources/features/alcoholToDeclareJourney/AlcoholDutyReturnFullJourney.feature:1366]: Ok..Then thats fine

**Review by Kavitakum — APPROVED** [review]
Approved

**Pull request merged**